seed-job/buildConfiguration.yaml

```diff
@@ -0,0 +1,17 @@
+builds:
+  - name: Rest-Gateway
```

Comment:
do you need - or can you use space?

Reply:
I guess need, as those are "array" entries, right?

Reply:
yes, its an array.

Reply:
i meant `Rest-Gateway` vs `Rest Gateway`